app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -288,7 +297,13 @@ class ZimManageViewModel @Inject constructor(
     Function6(::combineLibrarySources)
   )
     .doOnNext { libraryListIsRefreshing.postValue(false) }
+    .doOnError { throwable ->
+      if (throwable is OutOfMemoryError) {
+        Log.e("ZimManageViewModel", "Error----${throwable.printStackTrace()}")
+      }
+    }
     .subscribeOn(Schedulers.io())
+    .observeOn(AndroidSchedulers.mainThread())
```

Comment:
@CalebKL Why we are now starting to observe this on the main thread?

Reply:
I have made the changes to remove this line

Reply:
But I am still seeing this code.